doc/installation.rst

```diff
@@ -5,11 +5,15 @@ Installation Guide
 Spyder is relatively easy to install on Windows, Linux and macOS.
 Just make sure to read and follow these instructions with care.
 
-If you run into problems, before posting a report, *please* consult our comprehensive `Troubleshooting Guide`_ and search the `issue tracker`_ for your error message and problem description.
-These methods generally fix or isolate the great majority of install-related difficulties.
-Thanks!
+.. important::
+
+   Spyder now offers :ref:`standalone_installers_ref` for Windows and macOS, making it easy to get up and running with the IDE without having to download Anaconda or manually install it in your existing environment.
```

Comment:
```suggestion
   Spyder now offers :ref:`standalone_installers_ref` for Windows and macOS, making it easier to get up and running with the program without having to download Anaconda or manually install it in your existing environment.
```

* I think `easier` is better because it compares the current situation (with the installers) with the previous one (without them).
* I suggest changing `IDE` for `program` to avoid a technical term that newbies are unfamiliar with.

Reply:
```suggestion
   Spyder now offers :ref:`standalone_installers_ref` for Windows and macOS, making it easier to get up and running with the application without having to download Anaconda or manually install it in your existing environment.
```

> I think easier is better

Great thinking, agreed.

> I suggest changing IDE for program

Gotcha, I changed it to "application" since that's a more modern, specific and friendlier term.

Reply:
Sounds good to me. Please apply your suggestion instead.

Reply:
Done, thanks